src/common.build.pre.props

```diff
@@ -123,9 +123,9 @@
       <UseStandardPreprocessor>true</UseStandardPreprocessor>
       <LanguageStandard>stdcpp20</LanguageStandard>
       <LanguageStandard_C>stdc17</LanguageStandard_C>
-      <AdditionalOptions>%(AdditionalOptions) /utf-8 /Zc:externConstexpr /Zc:lambda /Zc:throwingNew</AdditionalOptions>
+      <!-- /fp:contract used to default to on before VS 17.0. If enabled it allows FMA for floats. -->
+      <AdditionalOptions>%(AdditionalOptions) /utf-8 /Zc:externConstexpr /Zc:lambda /Zc:throwingNew /fp:contract</AdditionalOptions>
       <ControlFlowGuard>Guard</ControlFlowGuard>
-      <FloatingPointModel>Fast</FloatingPointModel>

```

Comment:
My primary concern is with `/fp:fast` preventing `isnan` from working correctly under all circumstances.

Reply:
Do we need `nan`s to work properly? What circumstances? Where are we seeing specific issues?

Reply:
WinUI for instance uses NANs to signal empty NumberBoxes. Since we don't write a lot of floating point math I think there's almost no difference between the default behavior and `/fp:fast`. `ColorFix` might be the only code we have that benefits from it (and it benefits a lot).